### ISSUE TITLE
struct2depth: where is the evaluate.py?

### ISSUE THREAD

**Issue by TengFeiHan0** [tensorflow/models#7824]
I'm trying to evaluate the result received from this model, However, in this project, I don't find any code or file named `evaluate.py`. Does anyone know the reason?

OS Platform and Distribution: ubuntu  18.04
TensorFlow installed from: web
TensorFlow version: 2.0
Bazel version
CUDA/cuDNN version 7.5
GPU model and memory: 1080Ti

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by jimaldon** [comment]
The `README.md` in https://github.com/tensorflow/models/tree/master/research/struct2depth mentions the use of `evaluate.py` but it's nowhere to be found. @aneliaangelova

**Comment by DavidMorillaCabello** [comment]
Same here, looking for the evaluation script.

**Comment by aneliaangelova** [comment]
We use the evaluation code from the SFM learner, which is from Zhou et al.
https://github.com/tinghuiz/SfMLearner

**Comment by DavidMorillaCabello** [comment]
Thank you very much for your answer!

**Comment by aneliaangelova** [comment]
As mentioned above we use SfMLearner eval code (pls see above)

**Comment by mdfirman** [comment]
Many thanks for the pointers to SfMLearner evaluation code.

When evaluating on Cityscapes you set `eval_mode="cityscapes"`, but this isn’t an option in sfmlearner.
Could you please help me understand what is happening in this mode?  In particular:

- Are predicted depths capped to 80m and pixels with ground truth > 80 and < 1e-3 masked out, as in KITTI?
- Is evaluation done at full scale (2048x1024), or at the output scale of the model?
- I presume that as `eval_crop="none"` you aren't doing any cropping – is this right?
- Please could you share `test_files_cityscapes_triplets.txt` and the equivalent training txt file to clarify which training and test images you are using?
- And just to check – are you using the precomputed SGM disparities on the test set, converted to depth?
- Is there anything else I should know about the CS evaluation?

Sorry for the massive dump of questions, and thanks a lot for any help you can give.
